Lib/locale.py

```diff
@@ -217,10 +219,12 @@ def _format(percent, value, grouping=False, monetary=False, *additional):
             formatted = _strip_padding(formatted, seps)
     return formatted
 
-def format_string(f, val, grouping=False):
+def format_string(f, val, grouping=False, monetary=False):
     """Formats a string in the same way that the % formatting would use,
     but takes the current locale into account.
-    Grouping is applied if the third parameter is true."""
+    Grouping is applied if the third parameter is true.
```

Comment:
As long as we are editing this, I think there should be a blank line after the first sentence, before the 'Grouping' sentence.